app/screens/TodoList/layout.js

```diff
@@ -1,19 +1,24 @@
 import React from 'react';
-import { View } from 'react-native';
+import { View, StyleSheet, FlatList } from 'react-native';
 
-import TodoList from './components/TodoList';
+import TodoItemContainer from './components/TodoItem';
 import Header from './components/Header';
 import TodoInput from './components/TodoInput';
 import RemoveAllItems from './components/RemoveAllItems';
 
 function TodoListScreen({ onAddItem, listData, onDeleteItem, onRemoveAllItems }) {
+  renderList = item => (
```

Comment:
change the `TodoListScreen` from function to PureComponent since it hasnt state but need internal functions like the `renderList`
so, now render list should be an attribute of the `TodoListScreen` Pure class